src/firefly/js/ui/VersionInfo.jsx

```diff
@@ -52,14 +55,20 @@ function VersionInfoFull() {
         + (ffTag ? `\n${FF_TAG}:         ${ffTag}` : '');
 
     return (
-        <div style={{display: 'flex', justifyContent: 'space-evenly', flexDirection:'column'}}>
-            <div style={{display: 'flex', justifyContent: 'flex-start'}}>
-                <div style={{paddingRight:10, display: 'flex', flexDirection:'column', alignItems:'center', marginTop:-20}}>
-                    <a href='https://github.com/Caltech-IPAC/firefly' target='github-window'><img alt='Firefly' src={FFTOOLS_ICO} style={{marginTop:10}}/></a>
-                    <a href='https://www.caltech.edu' target='caltech-window'><img style={{width:70}} alt='Caltech' src={CALTECH_LOGO}/></a>
-                    <a href='https://www.ipac.caltech.edu' target='ipac-window'><img alt='IPAC' src={IPAC_LOGO}/></a>
-                </div>
-                <div style={{display: 'flex', justifyContent: 'space-evenly', marginLeft:60, alignItems:'flex-start'}}>
+        <Stack justifyContent='space-evenly' direction='column' sx={{fontSize: 14, width: '100%'} }>
+            <Stack direction='row'>
+                <Stack direction='column' alignItems='center' sx={{pr:0.6, mt:-2} }>
```

Comment:
- `pr:0.6` is not needed.
- `mt:-2` is to fix the `marginTop:10` set in its child img.  You can remove both instead.